iiop-openjdk/src/main/resources/org/wildfly/iiop/openjdk/LocalDescriptions.properties

```diff
@@ -26,11 +27,21 @@ iiop-openjdk.export-corbaloc=Indicates whether the root context should be export
 # security configuration properties.
 iiop-openjdk.support-ssl=Indicates whether SSL is to be supported (on) or not (off).
 iiop-openjdk.security-domain=The name of the security domain that holds the key and trust stores that will be used to establish SSL connections.
+iiop-openjdk.server-ssl-context=The name of the SSL context used to create server side SSL sockets.
+iiop-openjdk.client-ssl-context=The name of the SSL context used to create client side SSL sockets.
+iiop-openjdk.client-requires-ssl=Indicates whether IIOP connections from the server require SSL.
+iiop-openjdk.server-requires-ssl=Indicates whether IIOP connections to the server require SSL.
+iiop-openjdk.ssl-enabled-protocols=Protocols that will be used be SSL connections.
 iiop-openjdk.add-component-via-interceptor=Indicates whether SSL components should be added by an IOR interceptor (on) or not (off).
+iiop-openjdk.add-component-via-interceptor.deprecated=Indicates whether SSL components should be added by an IOR interceptor (on) or not (off).
```

Comment:
This and the other deprecated messages, should not be a copy of the original, but rather contain (short) information about the new way